src/node/status_node.py

```diff
@@ -76,6 +76,23 @@ def get_pubkey(self):
     def wait_fully_started(self):
         assert self.search_logs(string="retrieve messages...")
 
+    def wait_for_logs(self, strings=None, timeout=10):
+        if strings is not None and not isinstance(strings, list):
```

Comment:
The `strings is not None` is also covered by the `not isinstance(strings, list)`.